src/libpriv/rpmostree-util.h

```diff
@@ -60,6 +60,14 @@ template<typename T>
     throw std::runtime_error(msg.str());
   }
 
+// Convert a GError into a C++ exception.  Takes ownership of the error.
+static inline void 
```

Comment:
Indentation looks off here

Reply:
Honestly I'm not sure what indentation to use...the C++ stuff has messed up our style a lot here obviously.  The ones above I indented originally because they're `template<>` but this is just a regular function.

I thought about trying to introduce `clang-format` but...I think that'd be a lot of churn.

I did fix the `{` to be on the newline like the rest of the code.